rustler/src/resource.rs

```diff
@@ -145,9 +144,28 @@ where
         }
     }
 
+    pub fn make_binary<'env, F>(&self, env: Env<'env>, f: F) -> Binary<'env>
```

Comment:
I don't know what is the docs policy, but maybe worth adding an example specially because the API uses functions?

Reply:
Sure, I'll do that before "undrafting".